frontend/src/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,7 +1,60 @@
-type Props = {
-  onSubmit: () => void;
-};
+import { useForm } from 'react-hook-form';
+import { Link } from 'react-router-dom';
+import { UserSignInPayload } from 'common/types/types';
+import { AppRoute, ButtonType, DataStatus, InputType, UserSignInPayloadKey } from 'common/enums/enums';
+import { SigninSchema } from 'validation-schemas/validation-schemas';
+import { useAppSelector, useDispatch } from 'hooks/hooks';
+import { auth as authActions } from 'store/actions';
+import { getResolver } from 'helpers/form/form';
+import styles from '../sign-in-form/styles.module.scss';
+import logo from 'assets/img/logo-dark.svg';
+import { Button, Input } from '../common/common';
+
+const resolver = getResolver<UserSignInPayload>(SigninSchema);
+
+const SignIn: React.FC = () => {
+  const { register, handleSubmit } = useForm<UserSignInPayload>({ resolver });
+
+  const { authStatus } = useAppSelector(({ auth }) => ({
+    authStatus: auth.dataStatus,
+  }));
 
-const SignInForm: React.FC<Props> = () => <h1>Sign In ♒</h1>;
+  const dispatch = useDispatch();
```

Comment:
you don't need use it
pass submit cb through props (check the sign-up component)